README.md

```diff
@@ -19,80 +19,84 @@ There are two ways to install Lightning Web Components Recipes:
 
 1. Set up your environment. Follow the steps in the [Quick Start: Lightning Web Components](https://trailhead.salesforce.com/content/learn/projects/quick-start-lightning-web-components/) Trailhead project. The steps include:
 
--   Enable Dev Hub in your Trailhead Playground
--   Install Salesforce CLI
--   Install Visual Studio Code
--   Install the Visual Studio Code Salesforce extensions, including the Lightning Web Components extension
+    - Enable Dev Hub in your Trailhead Playground
+    - Install Salesforce CLI
+    - Install Visual Studio Code
+    - Install the Visual Studio Code Salesforce extensions, including the Lightning Web Components extension
 
-2. If you haven't already done so, authenticate with your hub org and provide it with an alias (**myhuborg** in the command below):
+1. If you haven't already done so, authenticate with your hub org and provide it with an alias (**myhuborg** in the command below):
 
-```
-sfdx force:auth:web:login -d -a myhuborg
-```
+    ```
+    sfdx force:auth:web:login -d -a myhuborg
+    ```
 
-3. Clone the lwc-recipes repository:
+1. Clone the lwc-recipes repository:
 
-```
-git clone https://github.com/trailheadapps/lwc-recipes
-cd lwc-recipes
-```
+    ```
+    git clone https://github.com/trailheadapps/lwc-recipes
+    cd lwc-recipes
+    ```
 
-4. Create a scratch org and provide it with an alias (**lwc-recipes** in the command below):
+1. Create a scratch org and provide it with an alias (**lwc-recipes** in the command below):
 
-```
-sfdx force:org:create -s -f config/project-scratch-def.json -a lwc-recipes
-```
+    ```
+    sfdx force:org:create -s -f config/project-scratch-def.json -a lwc-recipes
+    ```
 
-5. Push the app to your scratch org:
+1. Push the app to your scratch org:
 
-```
-sfdx force:source:push
-```
+    ```
+    sfdx force:source:push
+    ```
 
-6. Assign the **recipes** permission set to the default user:
+1. Assign the **recipes** permission set to the default user:
 
-```
-sfdx force:user:permset:assign -n recipes
-```
+    ```
+    sfdx force:user:permset:assign -n recipes
+    ```
 
-7. Load sample data:
+1. Import sample data:
 
-```
-sfdx force:data:tree:import --plan ./data/data-plan.json
-```
+    ```
+    sfdx force:data:tree:import -p ./data/data-plan.json
+    ```
 
-8. Open the scratch org:
+1. Open the scratch org:
 
-```
-sfdx force:org:open
-```
+    ```
+    sfdx force:org:open
+    ```
 
-9. In **Setup**, under **Themes and Branding**, activate the **Recipes Lite** or **Recipes Blue** theme.
+1. In **Setup**, under **Themes and Branding**, activate the **Recipes Lite** or **Recipes Blue** theme.
 
-10. In App Launcher, select the **LWC** app.
+1. In App Launcher, click View all then select the **LWC** app.
```

Comment:
Not to be a pain, but shouldn't we **bold** the words "View all"?

Reply:
Absolutely, fixed.